### PR TITLE
Update mixin for new histograms

### PR DESCRIPTION
**Pull request by SuperQ**
* [x] I added CHANGELOG entry for this change.
* [ ] Change is not relevant to the end user.

## Changes

Update store dashboard for summary to histogram changes.
* Followup to #6131
* Update changelog to note breaking changes.

### CODE REVIEW

File: examples/dashboards/store.json [hunk 1]
```diff
@@ -1677,7 +1677,7 @@
                      "step": 10
                   },
                   {
-                     "expr": "histogram_quantile(0.99, sum by (job, le) (rate(thanos_bucket_store_sent_chunk_size_bytes_bucket{job=~\"$job\"}[$__rate_interval])))",
+                     "expr": "histogram_quantile(0.50, sum by (job, le) (rate(thanos_bucket_store_sent_chunk_size_bytes_bucket{job=~\"$job\"}[$__rate_interval])))",
```
Comment (saswatamcode) [inline examples/dashboards/store.json:1680]: Do we need to change this quantile?

Comment (SuperQ) [reply examples/dashboards/store.json:1680]: Yes, this is a typo fix. If you look below at the `legendFormat` it's supposed to be p50.

Comment (saswatamcode) [reply examples/dashboards/store.json:1680]: Ack thanks! That makes sense!

**Review by saswatamcode — APPROVED** [review]
LGTM!

**Pull request merged**